draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -931,6 +933,73 @@ As noted in {{multiplexing}}, all RoQ streams and RoQ datagrams begin with a flo
 
 Because QUIC provides a capability to migrate connections for various reasons, including recovering from a path failure ({{Section 9 of !RFC9000}}), when a QUIC connection migrates, a RoQ sender has the opportunity to revisit decisions about which RTP packets are encapsulated in QUIC streams, and which RTP packets are encapsulated in QUIC DATAGRAMs. Again, RoQ receivers need to be prepared for this eventuality.
 
+## RTCP Considerations
+
+RTCP was originally defined to be used with UDP, which implies (1) that
+transmission timing is largely under the control of the application (limited
+buffering inside the stack) and (2) that the overhead, *avg_rtcp_size*, used to
+compute the RTCP transmission interval could be deterministically computed by
+adding the IP asnd UDP headers. Both change when carrying RTCP over QUIC and
+they change in different ways when using QUIC streams vs. QUIC datagrams.
+
+### RTCP over QUIC datagrams {#rtcp-over-datagrams}
+
+When sending RTCP packets in QUIC datagrams this implies that a packet may not
+be immediately transmitted as it is subject to queuing and multiplexing with RTP
+packets and subject to congestion control. This means that a sending timestamp
+added to an RTCP packet, e.g., in an SR packet, may differ in unforeseeable ways
+from the actual time when the packet gets sent into the network while these are
+usually fairly close to each other for RTP-over-UDP. Effectively, we have a
+application sending timestamp *t_a* and the network transmission timestamp
+*t_n*. Applications just have to be aware that RTCP does not measure the network
+level RTT but rather the application layer RTT.
```

Comment:
I'm not sure how this is different from RT(C)P over UDP. Isn't this last sentence also true for that?

Reply:
Well, in most cases when running RTP/RTCP directly over UDP the only extra latency is the time it takes from having entered the application layer timestamp until an IP/UDP packet with the payload can be created. That is normally quite short when one is running directly on UDP sockets.